ros/api/v1/recommendations.py

```diff
@@ -1,8 +1,8 @@
-from ros.lib.models import Rule, RhAccount, System, db, PerformanceProfile
-from ros.lib.utils import is_valid_uuid, identity
+from flask import request
 from ros.lib.config import INSTANCE_PRICE_UNIT
+from ros.lib.models import Rule, System, db, PerformanceProfile
 from flask_restful import Resource, abort, fields, marshal_with
-from flask import request
+from ros.lib.utils import is_valid_uuid, identity, system_ids_by_org_id
 
```

Comment:
Nitpick : I would prefer to edit existing lines instead of adding new ones for library changes. So the diff looks good in the git.